tests/flytekit/unit/common_tests/test_workflow.py

```diff
@@ -252,13 +248,13 @@ def my_list_task(wf_params, a, b):
     assert n.inputs[1].var == 'required'
     assert n.inputs[1].binding.scalar.primitive.integer == 10
 
-    # Test that launch plan ID ref is flexible
+    # Test that workflow is saved in the node
     w._id = 'fake'
     assert n.workflow_node.sub_workflow_ref == 'fake'
     w._id = None
 
     # Test that outputs are promised
-    n.assign_id_and_return('node-id')
+    n.assign_id_and_return('node-id*')  # dns'ified
```

Comment:
Whats this testing?

Reply:
that the dns filter strips it out i guess.